sdk/packages/analytics/src/e2e.test.ts

```diff
@@ -34,7 +34,7 @@ import {
 const TEST_CLOUD_ENDPOINT = 'http://127.0.0.1:6969/cloud'
 const TEST_ENDPOINT = 'http://127.0.0.1:6969/cloud/public/events'
 
-describe('Analytics client tests', () => {
+describe.concurrent('Analytics client tests', () => {
```

Comment:
do we want this?

Reply:
It would make it closer to the real use case, speeds up tests as well